components/common/FeedbackForm.js

```diff
@@ -73,6 +82,108 @@ const FeedbackForm = ({
     createUserFeedback({ variables: { input: data } });
   };
 
+  const renderAdditionalField = (field) => {
+    const requiredMessage = ` (${t('required-field')})`;
+
+    switch (field.fieldType) {
+      case 'text':
+        return (
+          <Controller
+            key={field.fieldLabel}
+            name={field.fieldLabel}
+            control={control}
+            defaultValue=""
+            rules={{
+              required: field.fieldRequired && t('error-field-required'),
+            }}
+            render={({ field: controllerField }) => (
+              <TextInput
+                {...controllerField}
+                id={`dynamic-${field.fieldLabel}`}
```

Comment:
We should avoid using ids with white space. Typically ids should just be alphanumeric, dashes and underscores.

I can't remember what happens, but it might not explode, so this can probably be sorted later if we don't have something like `field.fieldId` available for an easy fix.